placeholders.go

```diff
@@ -46,6 +47,18 @@ func parsePlaceholders(input string, r *http.Request) string {
 			}
 			input = strings.Replace(input, "{user}", user, -1)
 		}
+		if strings.HasPrefix(placeholder[0], "{label") {
+			nStr := placeholder[0][6 : len(placeholder[0])-1] // get the integer N in "{labelN}"
+			n, err := strconv.Atoi(nStr)
+			if err != nil || n < 1 {
+				input = strings.Replace(input, placeholder[0], "", -1)
+			}
+			labels := strings.Split(r.Host, ".")
+			if n > len(labels) {
+				input = strings.Replace(input, placeholder[0], "", -1)
```

Comment:
Why the decision to not error?
Not saying it should error here, just want to know more about the thought process.

Reply:
This is a bit different than the above issue.
When would we have a higher number `N` than labels?
Would this make sense to fail?
Could it be actually useful? Say if no subdomain of a subdomain is available it could fallback to returning an empty label.

The other option would be to error and let the actual fallback code handle it. In this case I think it comes down to being strict vs being a bit more forgiving.

Reply:
I'm not entirely sure when we'd have a higher number `N` than labels other than maybe a user error in the txt record. I think I need more information about use cases to decide whether it would be better to error or not here.